NguyenVanViet2004/FE-GlowUp
# Implement checkout UI

## Related issues
Resolved https://github.com/NguyenVanViet2004/FE-GlowUp/issues/31

## Summary
- Implement `Checkout` screen
- Add checkout to expo-stack
- Add new translation keys & color codes

## How to test
- navigate to `app/index.tsx` and replace 
 ```jsx
router.replace('/(tabs)/home')
```
with
```tsx
router.replace('/checkout/BookingCheckout')
```

- Verify the UI elements dark/light mode

## Concern points

## Screenshots
| Light mode | Dark mode |
| --- | --- |
|![Simulator Screenshot - iPhone 16 Plus - 2024-10-19 at 10 45 30](https://github.com/user-attachments/assets/f59a37ae-bae8-479a-9c87-c70106c8d6c0)|![Simulator Screenshot - iPhone 16 Plus - 2024-10-19 at 10 45 45](https://github.com/user-attachments/assets/b1d5114f-c9ac-4ab5-bee4-f8a12cfd069e)|
![Simulator Screenshot - iPhone 16 Plus - 2024-10-19 at 10 45 34](https://github.com/user-attachments/assets/8995b3b5-437b-4dbb-bbe2-f21133d99f96)|![Simulator Screenshot - iPhone 16 Plus - 2024-10-19 at 10 45 43](https://github.com/user-attachments/assets/f72e02eb-d16d-4fc7-a79a-f514b92c2943)|
